### PR TITLE
Android implementation

### PR DESCRIPTION
**Pull request by jeremie-movify**
Bound all of Qualtrics' SDK methods to plugin

### CODE REVIEW

**Comment by jeremie-movify** [comment]
@petermnt @timrijckaert @Kymer Please review the iOS implementation PR before this one (this branch is based off the ios-implementation branch)

File: android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt [hunk 1]
```diff
@@ -9,37 +11,98 @@ import io.flutter.plugin.common.MethodChannel.Result
...
+        }.exhaustive
+    }
+
+    private fun init(parameters: Extractor.QualtricsCall.Init, result: Result) {
+        Qualtrics.instance().initialize(parameters.brandId, parameters.zoneId, parameters.interceptId, context)
+        result.success(true)
+    }
+
+    private fun evaluateTargetingLogic(result: Result) {
+        Qualtrics.instance().evaluateTargetingLogic {
```
Comment (petermnt) [inline android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt:64]: maybe pass an error here

File: android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt [hunk 2]
```diff
@@ -9,37 +11,98 @@ import io.flutter.plugin.common.MethodChannel.Result
...
+            if (it.error != null) {
+                result.success(false)
+            }
+            else {
```
Comment (petermnt) [inline android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt]: else on same line :D

**Review by petermnt — CHANGES_REQUESTED** [review]

File: android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt (hunk 1 above)
Comment (jeremie-movify) [reply android/src/main/kotlin/be/vrt/qualtrics_flutter/QualtricsFlutterPlugin.kt:64]: 👍

**Review by petermnt — APPROVED** [review]

**Pull request merged**